_content/monographs/buddhas-sayings_ireland.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Itivuttaka: Buddha's Sayings"
+authors: ireland
```

Comment:
`authors` is an array. It needs `[]` around it :smile:

Reply:
I didn't understand this fix. Maybe you can explain it when we meet Sunday.